apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -159,6 +160,112 @@ function CopyScheduleButton() {
     );
 }
 
+interface CustomEventsBoxProps {
+    customEvents: RepeatingCustomEvent[];
+    isSkeletonMode: boolean;
+    scheduleNames: string[];
+}
+
+function CustomEventsBox(props: CustomEventsBoxProps) {
+    const { scheduleNames, isSkeletonMode } = props;
+
+    const [customEvents, setCustomEvents] = useState(props.customEvents);
```

Comment:
This was a stylistic choice to use `props.customEvents` instead of another prop name due to the overlap in naming with useState. 

Open to suggestions if this is too confusing / non-idiomatic.